.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Docker image
+
+on: push
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - name: Push to Github Container Registry
+        uses: docker/build-push-action@v2
+        with:
+            push: true
+            tags: ghcr.io/embarkstudios/cargo-deny-action:latest
```

Comment:
I think this might need to be examined, whether the image needs to have a different tag or anything.

Reply:
Yah, I think we would actually want exactly what [the docs](https://docs.github.com/en/free-pro-team@latest/actions/guides/publishing-docker-images#publishing-images-to-github-packages) have, only push on tagged releases, and use the release tag as the image tag.